REST/PowerShell/Projects/UpdateVCSCredsUrl.ps1

```diff
@@ -0,0 +1,25 @@
+$ErrorActionPreference = "Stop";
+
+# Define working variables
+$OCTOPUS_API_BASE_URL = "https://youroctourl"

```

Comment:
Could you match the variable syntax that you use for other variables here as well?